### PR TITLE
Fix more broken refs.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/users/prev_whats_new/whats_new_1.3.rst [hunk 1]
```diff
@@ -304,9 +304,9 @@ rcParam has been set, and will not retroactively affect already
 existing text objects.  This brings their behavior in line with most
 other rcParams.
 
-``savefig.jpeg_quality`` added to rcParams
-``````````````````````````````````````````
-rcParam value ``savefig.jpeg_quality`` was added so that the user can
+:rc:`savefig.jpeg_quality` added to rcParams
```
Comment (timhoffm) [inline doc/users/prev_whats_new/whats_new_1.3.rst]: Wouldn't do that in a heading. It's quite verbose due to mentioning the default.

https://9880-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/users/prev_whats_new/whats_new_1.3.html#rcparams-savefig-jpeg-quality-default-95-added-to-rcparams

Comment (anntzer) [reply doc/users/prev_whats_new/whats_new_1.3.rst]: What about
```
Added :rc:`savefig.quality`
```
?
I think the title reads well in that case ("Added rcParams["savefig.quality"] (default: 95)").

Comment (timhoffm) [reply doc/users/prev_whats_new/whats_new_1.3.rst]: Ok by me.

Comment (anntzer) [reply doc/users/prev_whats_new/whats_new_1.3.rst]: done

**Pull request merged**